src/openms/source/KERNEL/FeatureMap.cpp

```diff
@@ -126,14 +126,34 @@ namespace OpenMS
     unassigned_peptide_identifications_(source.unassigned_peptide_identifications_),
     data_processing_(source.data_processing_),
     id_data_() // updated below
-  {
-    // copy ID data and update references in features:
-    IdentificationData::RefTranslator trans = id_data_.merge(source.id_data_);
-    for (Feature& feature : *this)
     {
+      // copy ID data and update references in features:
+      IdentificationData::RefTranslator trans = id_data_.merge(source.id_data_);
+      for (Feature& feature : *this)
+      {
       feature.updateAllIDReferences(trans);
+      }
     }
-  }
+
+  FeatureMap::FeatureMap(FeatureMap&& source) :
+      Base(source),
+      MetaInfoInterface(source),
+      RangeManagerType(source),
+      DocumentIdentifier(source),
+      UniqueIdInterface(source),
+      UniqueIdIndexer<FeatureMap>(source),
+      protein_identifications_(source.protein_identifications_),
+      unassigned_peptide_identifications_(source.unassigned_peptide_identifications_),
+      data_processing_(source.data_processing_),
+      id_data_() // updated below
+      {
+        // copy ID data and update references in features:
```

Comment:
shouldn't the move constructor work without updating the references?

Reply:
e.g., does the default move work?

Reply:
Yes, seems to work.